src/app/features/fss-search/fss-search.component.ts

```diff
@@ -261,20 +261,27 @@ export class FssSearchComponent implements OnInit {
     configAttributes = AppConfigService.settings["fssConfig"].batchAttributes;
```

Comment:
The unit test `FssSearchComponent › should have filterGroups ordered to match the configuration after transformSearchAttributesToFilter is called`
is failing because `configAttributes` here is mocked incorrectly as an array of strings

Reply:
Resolved